Logger/FileLogger.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        public string LogFilePath { get; set; } = string.Empty;
```

Comment:
```suggestion
        public string LogFilePath { get; set; }
```

If you remove your default constructor on lines 12-15, you should be able to do the above and avoid a nullable error.